CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
+if(${BUILD_TESTING})
```

Comment:
These three lines should be deleted altogether, and the following should be placed below the `project` command:

```cmake
include(CTest)
```